utility_network/trace-a-utility-network/src/main/java/module-info.java

```diff
@@ -0,0 +1,9 @@
+module trace.a.utility.network.main {
+  requires com.esri.arcgisruntime;
+  requires javafx.base;
+  requires javafx.controls;
+  requires javafx.fxml;
+  requires javafx.graphics;
+  exports com.esri.samples.trace_a_utility_network;
+  opens com.esri.samples.trace_a_utility_network to javafx.fxml;
```

Comment:
I would see if removing `private` from the fields annotated with `@FXML` lets us remove the `opens`. The [doc](https://openjfx.io/javadoc/17/javafx.fxml/javafx/fxml/FXML.html) suggests they have to be `public` but I _think_ I have had it working when they have default scope. If we can get it to work like that it would avoid us having to provide an explanation as to why we need the `opens`.

Reply:
I was able to remove the `opens` by setting the fields to `public` as mentioned in the documentation. When using the default scope however, I encounter an inaccessible object error. Do let me know if there's a workaround for this or if I should keep the fields with `public` scope.

Reply:
@Luiy0 I think you'll need to get guidance from whoever is in charge of samples as to the path they want to take.

I would either
- make them `public` to avoid needing the `opens`
- add a comment to the `opens` explaining why it is required

Reply:
Got it, thanks @colinanderson.